File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int sum = (n*(n+1))/2;
        int par_sum = nums[0];
        for(int i=1; i<n ;i++){
            par_sum += nums[i];
        }
        return sum-par_sum; 
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        boolean[] check = new boolean[nums.length+1];
        for(int num : nums){
            check[num] = true;
        }
        for(int i=0;i<n;i++){
            if(!check[i]) return i;
        }
        return n;
    }
}